command/console.go

```diff
@@ -189,16 +189,15 @@ Usage: terraform [global options] console [options]
 
 Options:
 
-  -state=path            Path to read state. Defaults to "terraform.tfstate"
-
-  -var 'foo=bar'         Set a variable in the Terraform configuration. This
-                         flag can be set multiple times.
-
-  -var-file=foo          Set variables in the Terraform configuration from
-                         a file. If "terraform.tfvars" or any ".auto.tfvars"
-                         files are present, they will be automatically loaded.
+  -state=path       Legacy option for the local backend only. See the local
```

Comment:
Just because it looks a bit different here -- is this spacing correct in comparison to other help text?

Reply:
My understanding of the convention here is that we've typically lined things up so there's a gutter of four cells between the two columns, so I shrunk this down after noticing that there was a lot more space here than typical, which I guessed was because this help text was originally derived from another command (probably `plan`) which had longer options and we didn't reformat it at that time.

However, you've been adjusting these more recently so if you adopted a different convention then I'd like to follow it. Did you standardize on a fixed width for the first column, or something else?

Reply:
Nope, thanks for asking; I didn't pick a standard, but since we've been caught by some help-text-weirdness in the past, I thought to ask. I think making a documented standard (and then, tested) is a good idea for a future note!

Reply:
Cool, thanks for confirming that! This "convention" I mentioned of a four-cell gutter was already an inference on my part based on pre-existing code when I joined, so I'm not actually sure that this _was_ the original design intent and instead I've just been using this as _some_ model to keep things consistent as I've been updating these, but I agree it would be good to actually write down a style guide for stuff like this so that we're not each independently inferring our own sense of it.

I'm going to go ahead with this change for now in the absence of that, based on your response here, but I'll also make a note to prompt a conversation about an explicit CLI style guide (ideally one that can be objectively/mechanically tested, as you say) once we get past the tactical work for the forthcoming release.